.mergify.yml

```diff
@@ -25,17 +31,26 @@ pull_request_rules:
   - name: L10N - Auto Merge
     conditions:
       - and:
-        - files~=(strings.xml|l10n.toml)
+        - files~=^(l10n.toml|app/src/main/res/values[A-Za-z-]*/strings\.xml)$
+        # /!\ The line above doesn't prevent random files to be changed alongside
+        # l10n ones. That's why the additional condition exists below. For more
+        # information: https://docs.mergify.com/conditions/#how-to-match-lists
+        - -files~=^(?!(l10n.toml|app/src/main/res/values[A-Za-z-]*/strings\.xml)).+$
         - or:
           - and:
             - author=mozilla-l10n-automation-bot
             - base=main
-            - status-success=pr-complete
+            - head=import-l10n
+            - status-success=complete-pr
           - and:
             - author=github-actions[bot]
-            - base~=releases[_/].*
-            # Taskcluster doesn't run when PRs are created by github-actions[bot]
-            # because it's not considered a collaborator.
+            - base~=^releases[_/].*
+            - head~=^automation/sync-strings-\d+
+            - status-success=complete-push
```

Comment:
The rule has been tested on release branches in https://github.com/mozilla-releng/staging-fenix/pull/12.